src/ITProfessions.ts

```diff
@@ -0,0 +1,7 @@
+export const ITProfessions = [
```

Comment:
I'm not sure, if it's the best place for this file. I would rather move to some common folder, as we may want to use it in other places as well. But we can leave it for now here and decide later about it.